### DISCUSSION TITLE
6C71C604-0ED1-2359-9305-5E9B667055AD

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#197] [General]
# 6C71C604-0ED1-2359-9305-5E9B667055AD

This addon adds a semi-fictional texture to the planet Neptune

https://celestia.mobi/addon?item=6C71C604-0ED1-2359-9305-5E9B667055AD

**Comment by Zayedeisa2016** [comment]
hello my name is Zayed- i love this app
